libs/database/src/lib/services/authentication.service.v2.ts

```diff
@@ -36,7 +37,7 @@ import { Account, Credential, User } from '../schemas';
 @Injectable()
 export class AuthenticationServiceV2 {
   constructor(
-    @InjectModel('Account') private accountModel: Model<Account>,
+    @InjectModel('Account') public accountModel: Model<Account>,
```

Comment:
```suggestion
    @InjectModel('Account') private accountModel: Model<Account>,
```
ตรงนี้แนะนำให้ใช้เป็น private ดีกว่าไหมครับ ในส่วน unit test ให้ใช้ (service as any).xxx แบบนี้แทน